src/DynamoCoreWpf/Controls/InstalledPackagesControl.xaml

```diff
@@ -0,0 +1,295 @@
+                                    <Grid x:Name="PackageGrid"  >
+                                        <DockPanel HorizontalAlignment="Stretch"
+                                                       >
+                                            <StackPanel HorizontalAlignment="Left"
+                                                            Orientation="Horizontal"
+                                                            Margin="10,5,5,7">
+                                                <TextBlock Text="{Binding Path=Model.Name}"
+                                                               ToolTip="{Binding Path=Model.Name}"
+                                                               FontSize="11"
+                                                               Width="150"
+                                                               TextTrimming="CharacterEllipsis"
+                                                               Margin="15,0,10,0"
+                                                               Foreground="#E5E5E5"></TextBlock>
+                                                <TextBlock Text="{Binding Path=Model.VersionName}"
+                                                               MinWidth="50"
+                                                               FontSize="11"
+                                                               Foreground="#E5E5E5"></TextBlock>
+                                                <TextBlock Text="{x:Static p:Resources.InstalledPackageViewPendingInstallButton}"
+                                                               Visibility="{Binding Path=Model.EnableOldMarkedForUnistallState, Converter={StaticResource BooleanToVisibilityCollapsedConverter}}"
+                                                               FontSize="11"
+                                                               FontStyle="Italic"
+                                                               MinWidth="160"
+                                                               TextAlignment="Center"
+                                                               Foreground="Red"></TextBlock>
+                                            </StackPanel>
+                                            <StackPanel HorizontalAlignment="Right" Orientation="Horizontal" Width="Auto">
+                                                <Button Name="MoreButton"
+                                                                Click="MoreButton_OnClick"
+                                                                Width="16"
+                                                                Height="16"
+                                                                Margin="0,0,10,0"
+                                                                Style="{StaticResource FlatButtonStyle}">
+                                                    <Image Source="/DynamoCoreWpf;component/UI/Images/menu_16_16.png"/>
```

Comment:
this icon is stored differently from the others?

Reply:
It is only icons in `DynamoModern.xaml` that uses `pack`. Dynamo would not start without it. I don't know why.